ui/apps/platform/src/Containers/Vulnerabilities/utils/sortUtils.tsx

```diff
@@ -15,6 +15,10 @@ export const aggregateByCVSS: SortAggregate = {
     aggregateFunc: 'max',
 };
 
+export const aggregateByEPSS: SortAggregate = {
```

Comment:
Nit for later, not for now - IMO we should have these aggregate constants named by what they do "aggregateByMax" and not which column it is being used for.